FirebaseSessions/Tests/Unit/Mocks/MockApplicationInfo.swift

```diff
@@ -26,15 +26,19 @@ class MockApplicationInfo: ApplicationInfoProtocol {
 
   var osName: String = ""
 
+  var mccMNC: String = ""
+
   static let testAppID = "testAppID"
   static let testBundleID = "testBundleID"
   static let testSDKVersion = "testSDKVersion"
   static let testOSName = "ios"
+  static let testMCCMNC = "testMCCMNC"
```

Comment:
Can we do this as the proto structure like the NetworkConnectionInfo and see how this gets converted to a string as we want?

Reply:
Let me follow up with this and see if it makes sense to you